scene/register_scene_types.cpp

```diff
@@ -1041,14 +1041,14 @@ void register_scene_types() {
 	ClassDB::add_compatibility_class("RayShape", "SeparationRayShape3D");
 	ClassDB::add_compatibility_class("RayShape2D", "SeparationRayShape2D");
 	ClassDB::add_compatibility_class("RemoteTransform", "RemoteTransform3D");
-	ClassDB::add_compatibility_class("RigidBody", "RigidDynamicBody3D");
-	ClassDB::add_compatibility_class("RigidBody2D", "RigidDynamicBody2D");
+	ClassDB::add_compatibility_class("RigidBody", "RigidBody3D");
+	ClassDB::add_compatibility_class("RigidBody2D", "RigidBody2D");
```

Comment:
Same name, not needed.

On the other hand, in the "alpha compatibility" section below you should add compat renames for the RigidDynamicBody -> RigidBody re-rename, or people's 4.0 alpha scenes will break.